tests/python_functional/src/message_builder/log_message.py

```diff
@@ -30,7 +30,7 @@ def __init__(self):
         self.timestamp_value = time.time()
         self.bsd_timestamp_value = "Feb 11 21:27:22"
         self.iso_timestamp_value = "2019-02-11T21:27:22+01:00"
-        self.hostname_value = socket.gethostname()
+        self.hostname_value = socket.gethostname().split('.')[0]
         self.program_value = "testprogram"
```

Comment:
ditto, maybe @mitzkia has an opinion too.

Reply:
Answered in a previous comment.